tritond/main_test.go

```diff
@@ -80,11 +81,11 @@ func (c *consumer) Start() error {
 			}
 		}
 	}()
+	runtime.Gosched()
 
 	return nil
 }
 
-func (c *consumer) Stop() error {
+func (c *consumer) Stop()  {
 	close(c.close)
```

Comment:
This can also be solved by requiring callers to do this on the same goroutine right?

Reply:
No. The problem is that the read happens on the different one. So close must happen on the same one that the read and create happens on.

Reply:
Realized this is only for the test client so this is much less important